dwh/entrypoint.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This script is the entry-point to any indicator calculation logic.
+print_prefix="INDICATOR GENERATION:"
+echo "${print_prefix} Transformation STARTED..."
+TEMP_OUT=$(mktemp  /dwh/output/${OUTPUT_CSV})
```

Comment:
What happens if the provided `OUTPUT_CSV` does not have the required `XXXXXX`? Does the script exit here? On a related note, should we `set -e` at the top of the script?

Reply:
Forgot about this comment! It fails with an error:  `mktemp: too few X's in template ‘/dwh/output/indicators_output.csv’` To avoid this error, please see the latest changes. I have also added `set -e` at the beginning